perllib/FixMyStreet/Cobrand/Bexley/Waste.pm

```diff
@@ -353,10 +353,20 @@ sub _in_cab_logs {
     my $dt_from = $self->_subtract_working_days(WORKING_DAYS_WINDOW);
     my $cab_logs;
     if ( !$self->{c}->stash->{cab_logs} ) {
-        $cab_logs = $self->whitespace->GetInCabLogsByUprn(
+        my $cab_logs_uprn = $self->whitespace->GetInCabLogsByUprn(
             $property->{uprn},
             $dt_from->stringify,
         );
+        my $cab_logs_usrn = $self->whitespace->GetInCabLogsByUsrn(
+            $cab_logs_uprn->[0]{Usrn},
+            $dt_from->stringify,
+        );
+        $cab_logs = [ @$cab_logs_uprn, @$cab_logs_usrn ];
+
+        # Make cab logs unique by LogID
+        my %seen;
+        @$cab_logs = grep { !$seen{ $_->{LogID} }++ } @$cab_logs;
```

Comment:
I should remember this as a pithy way to get unique elements.

Reply:
Always takes my brain a moment to decode this construct when I see it, so I'm never sure if it's straying into code-thats-a-bit-too-clever territory! As you say, it's a fairly brief way of getting this done (though IMO not as nice as Ruby's `cab_logs = cab_logs.uniq { |log| log[:LogID] }`!)